include/git2/version.h

```diff
@@ -13,6 +13,6 @@
 #define LIBGIT2_VER_REVISION 0
 #define LIBGIT2_VER_PATCH 0
 
-#define LIBGIT2_SOVERSION "1.2"
+#define LIBGIT2_SOVERSION "1"
```

Comment:
Why would we change the soversion from `1.2` to `1`?  Even if we stop updating the soversion on new releases, we're not going to go backward are we?

Reply:
Yes, your are clearly true!